.github/workflows/ci.yml

```diff
@@ -61,8 +61,6 @@ jobs:
         env:
           PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
           PGP_SECRET: ${{ secrets.PGP_SECRET }}
-          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
```

Comment:
This should stay, otherwise not password and username would be available. The release should be fixed, it seems org secrets were not set for this repo, which is why it didn't see the change

Reply:
reverted